wpilibj/src/main/java/edu/wpi/first/wpilibj/ADIS16448_IMU.java

```diff
@@ -1042,16 +1084,36 @@ public IMUAxis getYawAxis() {
     return m_yaw_axis;
   }
 
+  /**
+   * Returns the raw (un-offset) orientation of the device as a Rotation3d
+   *
+   * @return Rotation3d representing the device orientation
+   */
+  private synchronized Rotation3d getGyroOrientation() {
+    Rotation3d m_orientation;
```

Comment:
Nit: This local variable shouldn't use `m_x` notation. (Should either use just `orientation` or directly return the value inside the if statement)